projects/js-packages/connection/hooks/use-product-checkout-workflow/index.jsx

```diff
@@ -39,6 +40,17 @@ export default function useProductCheckoutWorkflow( {
 		from,
 	} );
 
+	const initializeAnalytics = () => {
+		const tracksUser = syncSelect( STORE_ID ).getWpcomUser();
+		const blogId = syncSelect( STORE_ID ).getBlogId();
```

Comment:
why not use the useSelect() hook?

Reply:
> Alias for the select built-in control in the @wordpress/data package.
https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data-controls/#syncselect

Reply:
Seems to be tagged as deprecated though

Reply:
What's [deprecated is where the hook lives](https://github.com/WordPress/gutenberg/blob/trunk/packages/data-controls/CHANGELOG.md/#deprecations). It was moved from `@wordpress/data-controls` to `@wordpress/data` so it's OK to use since it's imported from the data pkg.

Reply:
Curious: what do we gain by using the hook instead of the synchronous select function?

Reply:
I think it's mostly a usage convention.  I see we use `useSelect()` instead of `select()` in the context of the hook.